src/actions/secrets.js

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { getCredentials, deleteCredential, createCredential } from '../api';
+import { getSelectedNamespace } from '../reducers';
+
+export function fetchSecretsSuccess(data, namespace) {
+  return {
+    type: 'SECRETS_FETCH_SUCCESS',
+    data,
+    namespace
```

Comment:
As mentioned previously this is not compatible with the 'all namespaces' option that will be enabled in the navigation very soon (WIP PR https://github.com/tektoncd/dashboard/pull/213)

Reply:
I added a commented version of 'all namespaces' below to be enabled once that PR rolls out.

Reply:
Currently credentials don't have a namespace field specified in the object struct in the backend. After going through your feedback i'll go ahead & added that code.